modules/infrastructure-components.adoc

```diff
@@ -1,21 +1,21 @@
 // Module included in the following assemblies:
 //
 // * machine_management/creating-infrastructure-machinesets.adoc
-// * post_installation_configuration/cluster-tasks.adoc
 // * nodes-nodes-creating-infrastructure-nodes.adoc
 
 [id="infrastructure-components_{context}"]
 = {product-title} infrastructure components
 
-The following infrastructure workloads do not incur {product-title} worker subscriptions:
+Each self-managed Red Hat OpenShift subscription includes entitlements for {product-title} and other OpenShift-related components. These entitlements are included for running {product-title} control plane and infrastructure workloads and do not need to be accounted for during sizing.
 
-* Kubernetes and {product-title} control plane services that run on masters
+To qualify as an infrastructure node and use the included entitlement, only components that are supporting the cluster, and not part of an end-user application, can run on those instances. Examples include:
```

Comment:
[Per the ISG](https://www.ibm.com/docs/en/ibm-style?topic=format-lists#lead-in-wording) we should avoid introducing lists without using a complete sentence, I'd suggest rewriting to something like the following:
```suggestion
To qualify as an infrastructure node and use the included entitlement, only components that are supporting the cluster, and not part of an end-user application, can run on those instances. Examples include the following components:
```
Feel free to use some other wording if it makes more sense, as long as it avoids lead in wording.